hknweb/candidate/views.py

```diff
@@ -76,17 +140,68 @@ def officer_confirm_view(request, pk):
     if form.is_valid():
         form.instance.reviewed = True
         form.save()
-        return redirect('/cand/dummy')
+        send_cand_confirm_email(form)
+        return redirect('/cand/reviewconfirm/' + str(pk))
```

Comment:
PLUS???

Ok I think this should be `'/cand/reviewconfirm/{}'.format(pk)` but that's just me crusading against Javascript.

Reply:
What does this have to do with javascript?

Reply:
Nothing tbh, JavaScript adds strings in weird ways